x/liquidgov/keeper/icqcallbacks_mirror_proposals.go

```diff
@@ -27,27 +25,29 @@ func MirrorProposalsCallback(k Keeper, ctx sdk.Context, args []byte, query icqty
 
 	highestID, _ := k.GetProposalID(ctx, chainId)
 
-	var proposals govtypesv1.Proposals
-	codec.NewLegacyAmino().UnmarshalJSON(args, &proposals)
-
-	// if err != nil {
-	// 	return sdkerrors.Wrapf(stakeibctypes.ErrMarshalFailure, "unable to unmarshal query response into Proposal type, err: %s", err.Error())
-	// }
+	queriedProposal := govtypesv1beta1.Proposal{}
```

Comment:
We may have to branch this by SDK version on the host chain (if the gov module was update between SDK 45/46)

Reply:
Not just host chain version, how to handle custom proposals of custom module on the zone? (Could we decode this and show this? Or should we just fetch proposal ID and the content is shown on frontend side?)

Reply:
Are there many custom proposal types across cosmos? Imo we should start with the basic code gov proposal types, then we can add the others in a future ugrade